gradle.properties

```diff
@@ -9,7 +9,7 @@ downloadPluginVersion=4.0.4
 releasePluginVersion=2.6.0
 ballerinaGradlePluginVersion=1.0.0
 
-ballerinaLangVersion=2201.8.0-20230726-145300-b2bdf796
+ballerinaLangVersion=2201.7.0
```

Comment:
Arent we going to move to java17 with the release?

Reply:
@gabilang will send a PR